scheduler/pkg/klcpermit/workflow_manager.go

```diff
@@ -76,7 +76,7 @@ func (sMgr *WorkloadManager) Permit(ctx context.Context, pod *corev1.Pod) Status
 	crd, err := sMgr.GetCRD(ctx, pod.Namespace, name)
 
 	if err != nil {
-		klog.Infof("[Keptn Permit Plugin] could not find workloadVersion crd %s, err:%s", name, err.Error())
+		klog.Infof("[Keptn Permit Plugin] could not find workloadVersion crd %s, err:%s", name, err.Error(), pod.GetObjectMeta().GetName(), pod.GetObjectMeta().GetNamespace())
```

Comment:
The name is already in the log message, why are we adding it again ?

Reply:
It mistake from my end I will go ahead and remove the name i added and leave only the namespace.

Reply:
Made the fix